.vscode/tasks.json

```diff
@@ -18,7 +18,7 @@
         {
             "label": "Configure",
             "type": "shell",
-            "command": "./configure --enable-ccache  --enable-experimental-plugins --enable-example-plugins --enable-test-tools --enable-debug --enable-werror",
+            "command": "autoreconf -if; ./configure --enable-ccache  --enable-experimental-plugins --enable-example-plugins --enable-test-tools --enable-debug --enable-werror",
```

Comment:
A minor thing, but I suggest `&&` to separate the commands in case autoreconf fails for some reason.

Reply:
👍 Yeah, I decided to do that and a few more things in the lasted commit.